Hospital-System/Hospital.Email/Services/EmailSender.cs

```diff
@@ -0,0 +1,38 @@
+﻿using FluentEmail.Core;
+using Hospital.Email.Base;
+using Hospital.Email.Services.Interfaces;
+using HospitalSystem.Common.Models.Configs;
+
+namespace Hospital.Email.Services;
+
+public class EmailSender : IEmailSender
+{
+    
+    private readonly IFluentEmail _fluentEmail;
+    private readonly EmailConfig _emailConfig;
+
+    public EmailSender(IFluentEmail fluentEmail, EmailConfig emailConfig)
+    {
+        _fluentEmail = fluentEmail;
+        _emailConfig = emailConfig;
+    }
+    
+    public async Task<bool> SendEmailAsync<T>(string to, T massage) where T : EmailMassageBase
+    {
+        var path = $@"{_emailConfig.TamplatePath}\{massage.TemplateName}.cshtml";
+        
+        try
+        {
+            var emailResponse = await _fluentEmail
+                .To(to)
```

Comment:
You have to validate `to` parameter. At least you have to check that it's actually an email address